src/outbox/outboxActions.js

```diff
@@ -12,7 +12,11 @@ import {
 } from '../actionConstants';
 import { getAuth } from '../selectors';
 import { sendMessage } from '../api';
-import { getSelfUserDetail, getUserByEmail } from '../users/userSelectors';
+import {
+  getSelfUserDetail,
+  getUserByEmail,
+  getAllUserAutocompleteOptions,
```

Comment:
This does not seem to be an appropriate name. Think of something else? (it is not only about autocomplete)

Reply:
Done 👍